OneCommunityGlobal/HighestGoodNetworkApp
# Shihao - BadgeImage Unit Test

# Description
Unite test for `BadgeImage.jsx`

## Main changes explained:
- Add the unit test

## How to test:
1. check into current branch
2. do npm install
3. do `npm test BadgeImage.test.js` to run the test
4. Verify that all 4 tests have passed

## Screenshots or videos of changes:
<img width="686" alt="Screenshot 2023-11-30 at 15 12 18" src="https://github.com/OneCommunityGlobal/HighestGoodNetworkApp/assets/62367016/eadda557-a40e-4354-9cd2-8eab235d82a1">
